projects/cmake/CMakeLists.txt

```diff
@@ -1,6 +1,9 @@
 cmake_minimum_required(VERSION 3.0)
 project(hebi_cpp_examples)
 
+SET (CMAKE_CXX_STANDARD 11)
```

Comment:
I would prefer for this to be set on each individual target, but it's not a big deal

Reply:
I'm fine either way -- this was just a quick addition because `develop` wasn't compiling for me after the last CMake changes you made.